website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -20,7 +20,7 @@ You can manage run tasks through the UI or through the [Run Tasks APIs](/cloud-d
 
 ## Creating a Run Task
 
-Refer to [Run Tasks Integration](/cloud-docs/integrations/run-tasks#hcp-packer-run-task) for a list of available run tasks. These include the HCP Packer integration and run tasks developed by HashiCorp Technology Partners.
+You can access a complete list of all run tasks in the [Terraform Registry](https://registry.terraform.io/browse/run-tasks). These run tasks include the [HCP Packer integration]((/cloud-docs/integrations/run-tasks#hcp-packer-run-task) and run tasks developed by HashiCorp Technology Partners.
```

Comment:
Wanted to make this active voice (Just saw this in my final review!) ok by you @gautambaghel ?
```suggestion
You can access a complete list of all run tasks in the [Terraform Registry](https://registry.terraform.io/browse/run-tasks). These run tasks include the [HCP Packer integration]((/cloud-docs/integrations/run-tasks#hcp-packer-run-task) and run tasks that our HashiCorp Technology Partners develop and maintain.
```

Reply:
yes, perfectly fine with me.